contracts/Interfaces/IUniswapV2Factory.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity >=0.5.0;
+// SPDX-License-Identifier: MIT
+
+interface IUniswapV2Factory {
```

Comment:
as this interface not use in our source code ,so I suggest add abi json file to migration/helpers and use it in migrations scripts , and please remove it

Reply:
@c-future I tried to do this in [this commit](https://github.com/veTokenFinance/veToken-contracts/pull/4/commits/365d6427b838d84e17172f6cf44fb171d77f6e2b) but for some reason the transaction for creating a pair is failing now while it worked before using artifacts. Do you know what went wrong here perhaps?

Reply:
@c-future I have fixed this issue now